### ISSUE TITLE
Broken as of 2:30 pm EDT July 22 2024

### ISSUE THREAD

**Issue by eberkopec** [yuliskov/SmartTube#3744]
no videos were working when I last attempted to watch. I got the error 402 message, and then nothing played on any channel.

**Comment by Patraputt** [comment]
Smarttube is fully broken.

**Comment by nadavf24** [comment]
Yup broken here too. Unable to load anything. No toast for error code as well.

**Comment by cycaboose** [comment]
Same here, no error messages or toast, when I open a video it just doesn't play.  Tried clearing cache, still no joy

**Comment by ihauh** [comment]
I suggest trying to play several videos and the same video a few times. Then they started playing for me with the latest update.

**Comment by eberkopec** [comment]
After update 22.60? I just tried and I’m still not having luck. Maybe I’m just not patient enough. I will keep trying. Thank you for the message.Sent from my iPhoneOn Jul 22, 2024, at 3:53 PM, ihauh ***@***.***> wrote:﻿
I suggest trying to play several videos and the same video a few times. Then they started playing for me with the latest update.

**Comment by philshel1** [comment]
Worked for me as well.  I kept opening different videos until one ran, then they all did.

**Comment by eberkopec** [comment]
How long did it take you?Sent from my iPhoneOn Jul 22, 2024, at 4:05 PM, philshel1 ***@***.***> wrote:﻿
Worked for me as well.  I kept opening different videos until one ran, then they all did.

**Comment by aide199** [comment]
Videos are now playing on 22.60 but all 4k videos are only plating at 1080. 

Anyone else having the same issuw with 22.60?

**Comment by eberkopec** [comment]
I am having nothing play at all with 22.60. I even forced stopped and cleared the cache. Nothing is playing at all.Sent from my iPhoneOn Jul 22, 2024, at 4:12 PM, aide199 ***@***.***> wrote:﻿
Videos are now playing on 22.60 but all 4k videos are only plating at 1080.
Anyone else having the same issuw with 22.60?

**Comment by philshel1** [comment]
> How long did it take you?

Only 5 vids.

**Comment by busybit** [comment]
No videos playing on Google Chromecast.

**Comment by Joshuawilliamson** [comment]
NVidia Shield, using version 22.44
No videos play, even after attempting 10 times.

**Comment by philshel1** [comment]
> Videos are now playing on 22.60 but all 4k videos are only plating at 1080.
> 
> Anyone else having the same issuw with 22.60?

Same here, got it to play but no 4k.

**Comment by eberkopec** [comment]
Well, I tried more than five. No luck. I’ll just have to wait and see what happens.Sent from my iPhoneOn Jul 22, 2024, at 4:24 PM, philshel1 ***@***.***> wrote:﻿

How long did it take you?

Only 5 vids.

**Comment by mgrundnig** [comment]
I have the same issue with 22.60.
No videos play, tried to play a bunch of different videos, but no luck.

**Comment by omzig10** [comment]
> NVidia Shield, using version 22.44 No videos play, even after attempting 10 times.

Same.

**Comment by mrfhenix** [comment]
i have youtube premium would you find it Useful to have some LOGS or something
to find a loop hole .??  every version works with a premium account probably we can help find a way to fix this issue

**Comment by snuffomega** [comment]
**Workaround** Turn Off SponsorBlock

Working on - a firetv & googletv. Hopefully this helps others and dev with the current snafu.

**Comment by Joshuawilliamson** [comment]
> **Workaround** Turn Off SponsorBlock
> 
> Working on - a firetv & googletv. Hopefully this helps others and dev with the current snafu.

Turning off Sponsor Block does not work for me.   v22.44  YouTube Premium on Nvidia Shield.  Sponsor Blocks servers show up.

**Comment by Kicco77** [comment]
> > **Workaround** Turn Off SponsorBlock
> > Working on - a firetv & googletv. Hopefully this helps others and dev with the current snafu.
> 
> Turning off Sponsor Block does not work for me. v22.44 YouTube Premium on Nvidia Shield. Sponsor Blocks servers show up.

Same here on 22.61 …

**Comment by Kicco77** [comment]
Update 22.62 Nvidia Shield … videos still don’t play 🥲

**Comment by Joshuawilliamson** [comment]
> Update 22.62 Nvidia Shield … videos still don’t play 🥲

I'm not on SmartTube Beta, just using the normal one. 
Checking for update, it says I have the latest one at 22.44
SHIELD Android TV S Ver:  9.1.1(33.2.0.157)   Kernal ver: 4.8.141

**Comment by FolkSong** [comment]
I'm on 22.41 as well. I can confirm the latest beta fixes it. But it's kind of annoying because it installs as a separate app, so all my settings are gone.

**Comment by omzig10** [comment]
Working great right now on 22.65... on a Shield TV.

**Comment by Joshuawilliamson** [comment]
Can confirm, version 22.65 fixes the issue with my Nvidia Shield. 
Thanks everyone!

**Comment by busybit** [comment]
22.65 also works on Chromecast now.

**Comment by aitronssesin** [comment]
I'm on 22.65 in Google TV (Chromecast) and it doesn't work

**Comment by Katerle** [comment]
Short Feedback: On 22.65 Beta (FireTV 4K, Loged in User) the 403 error seemst to be gone. 4K I hav'nt tried...

**Comment by Patraputt** [comment]
Everyone, please update to 22.65. It'll work like a charm!
